roles/klaytn_node/tasks/run_klaytn_tools.yml

```diff
@@ -0,0 +1,215 @@
+---
+# description: Download/configure/run Klaytn tools - homi and kgen
+- name: Klaytn Homi Tool - Create Homi Directory
+  file:
+    path: "{{ homi_root }}/homi"
+    state: directory
+    mode: "0755"
+  when:
+    - not remoteNode|bool
+
+- name: Klaytn Homi Tool - Download Package
+  get_url:
+    url: "http://packages.klaytn.net/klaytn/{{ klaytn_homi.version }}/homi-{{klaytn_homi.version }}-0-linux-amd64.tar.gz"
```

Comment:
In this case, Can you consider other os, too?

Reply:
Such change is being reviewed in #8.